### ISSUE TITLE
/mnt/container_volumes is read only on DigitalOcean

### ISSUE THREAD

**Issue by siddharthist** [CiscoCloud/microservices-infrastructure#849]
When I try to store anything in the `/mnt/container-volumes` folder, I get this message on DigitalOcean:
```
[root@lb-mi-worker-003 ~]# mkdir /mnt/container-volumes/test
mkdir: cannot create directory ‘/mnt/container-volumes/test’: Read-only file system
```
I think this may be because GlusterFS is not supported on DO. The docs mention "supported clouds", but I can't find a list anywhere. If this is the case, then we should document which clouds support Gluster on [this page](http://microservices-infrastructure.readthedocs.org/en/latest/components/glusterfs.html).

**Comment by BrianHicks** [comment]
That shouldn't be happening, actually... did the GlusterFS role complete successfully?

**Comment by siddharthist** [comment]
Yep!

**Comment by eirslett** [comment]
Maybe there should be a distributive test for that as well? Create a tmp directory and see if it works?

**Comment by ryane** [comment]
this is not limited to digital ocean. I am currently seeing the same on AWS

**Comment by BrianHicks** [comment]
This appears to be fixed in #880.

**Comment by ryane** [comment]
fixed by #880
